File: Translation_cf.py
a=str(input())
b=str(input())
c=''.join(reversed(a))
n=min(len(b),len(a))
check=0
for i in range(0,n):
    if c[i]!=b[i]:
        check=1
if check==0:
    print("YES")
else:
    print("NO")

File: Translation_cf.cpp
#include <bits/stdc++.h>
using namespace std;
string a;
string b;
int check = 0;
int main()
{

    cin >> a >> b;
    string c = "";
    for (int i = a.length() - 1; i >= 0; i--)
        c = c + a[i];
    for (int i = 0; i < a.length(); i++)
        if (c[i] != b[i])
            check = 1;
    if (check == 0)
        cout << "YES";
    else
        cout << "NO";
    return 0;
}